### PR TITLE
Switch to store in suggestions

### PR DESCRIPTION
**Pull request by alshakerM**
Changed how we get the data in suggestions from React.useEffect fetch to store.

### CODE REVIEW

**Comment by alshakerM** [comment]
Deleted console logs in resolvers.

File: src/Suggestions/Suggestions.js [hunk 1]
```diff
@@ -3,16 +3,13 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { Avatar } from '../Avatar/Avatar';
 import styles from './Suggestions.module.css';
+import {  useSelect } from '@wordpress/data';
```
Comment (alshakero) [inline src/Suggestions/Suggestions.js:6]:
```suggestion
import { useSelect } from '@wordpress/data';
```

**Review by alshakero — APPROVED** [review]
Clean as a whistle ✨

File: src/Suggestions/Suggestions.js (hunk 1 above)
Comment (alshakerM) [reply src/Suggestions/Suggestions.js]: Done

**Pull request merged**